src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -738,7 +738,6 @@ class DocBaseViewer extends BaseViewer {
             disableFontFace,
             disableRange,
             disableStream,
```

Comment:
I would leave isEvalSupported:false here. It will protect us from any other vulnerabilities that might popup because of this. It's a supported parameter and it should always be false. Did the default value change to false in pdf.js?